StripeUICore/StripeUICore/Source/Controls/OneTimeCodeTextField.swift

```diff
@@ -838,7 +838,7 @@ private extension OneTimeCodeTextField {
             borderLayer.backgroundColor = backgroundColor.resolvedColor(with: traitCollection).cgColor
             borderLayer.borderColor = theme.colors.border.resolvedColor(with: traitCollection).cgColor
             caret.backgroundColor = label.textColor.resolvedColor(with: traitCollection).cgColor
-            focusRing.borderColor = tintColor.cgColor
+            focusRing.borderColor = tintColor.resolvedColor(with: traitCollection).cgColor
```

Comment:
The values changed here were previously not respecting forced override of user interface style. This fixes the issue, which should be evident in the screenshots.